addons/web/static/src/js/views/list/list_model.js

```diff
@@ -42,7 +42,7 @@
          * @param {string[]} recordIds a list of record datapoint ids
          * @param {string} fieldName the field to write
          */
-        saveRecords: function (listDatapointId, referenceRecordId, recordIds, fieldName) {
+        saveRecords: function (listDatapointId, referenceRecordId, recordIds, listFieldName) {
```

Comment:
listFieldNames (plural) ?
This seems like a change in API, (expecting a list of strings rather than one string)
So control where this is called

Reply:
Oups, indeed, it will change the name.
Yeah this method is only used `_saveMultipleRecords` method of `ListController` and there is no override. Thus, I think I can change the signature of this method 🙂 
If it is ok for you, I will update the docstring for this method too.